azext_iot/iothub/commands_device_messaging.py

```diff
@@ -0,0 +1,221 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.iothub.providers.device_messaging import DeviceMessagingProvider
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def iot_device_send_message(
+    cmd,
+    device_id: str,
+    data: str = "Ping from Az CLI IoT Extension",
+    properties: str = None,
+    msg_count: int = 1,
+    device_symmetric_key: str = None,
+    certificate_file: str = None,
+    key_file: str = None,
+    passphrase: str = "",
```

Comment:
None doesn't work as a default to passphrase?

Reply:
it does work - fixing